cells/std/cli/randomitems.go

```diff
@@ -0,0 +1,273 @@
+package main
+
+import (
+	"math/rand"
+	"sync"
+)
+
+type randomItemGenerator struct {
+	names          []string
+	nameIndex      int
+	organelles     []string
+	organelleIndex int
+	cells          []string
+	cellIndex      int
+	clades         []string
+	cladeIndex     int
+	descs          []string
+	descIndex      int
```

Comment:
and here an appropriate implementarion for generating dummy simple markdowm readmes